en/components/combo_cascading.md

```diff
@@ -1,90 +1,66 @@
 ---
 title: Cascading Scenario
-_description: The igx-combo and igx-drop-down exposes events, properties and methods that allows to easily construct cascading scenario using one of the controls or both of them.
-_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Combo components, Angular Combo controls, Angular Cascading Combo, Angular Drop Down, Angular Cascading Scenario
+_description: The igx-combo and igx-select exposes events, properties and methods that allows to easily construct cascading scenario using one of the controls or both of them.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Combo components, Angular Combo controls, Angular Cascading Combo, Angular Select, Angular Cascading Scenario
+_language: kr
 ---
 
 ## Cascading Scenario
 <p class="highlight">
 
-The [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) and [igx-drop-down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) exposes events, properties and methods that allows to easily construct cascading scenario, either using one of the controls or both of them.
+The [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) and [igx-select]({environment:angularApiUrl}/classes/igxselectcomponent.html) exposes events, properties and methods that allows to easily construct cascading scenario, either using one of the controls or both of them.
```

Comment:
Some small changes, since the point talks about both components
```suggestion
The [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) and [igx-select]({environment:angularApiUrl}/classes/igxselectcomponent.html) expose events, properties and methods that allow users to easily construct cascading scenarios, either using one of the controls or both of them.
```